package.json

```diff
@@ -59,6 +59,7 @@
     "redux-thunk": "^2.2.0",
     "reselect": "^2.5.4",
     "sha1": "^1.1.1",
+    "truffle-contract": "^3.0.0",
```

Comment:
Can you remove this change? I'm afraid it will break something if we simply deploy with it now

Reply:
@andre-meyer yes, sure. My local dev build wouldn't compile without this for some reason, thought i had to add it.